### PR TITLE
Remove conda forge testing in release documentation

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -1,7 +1,18 @@
 =========
 Changelog
 =========
+.. Future Release
```
Comment (thehomebrewnerd) [inline docs/source/changelog.rst:4]: Uncomment this. Also if we are going to use the underline approach for the release version/date like we do in other repos, should we update the past entries so the formatting is consistent? Not a big deal either way though.

Comment (rwedge) [inline docs/source/changelog.rst:4]: this should be uncommented since the release already happened

**Review by thehomebrewnerd — DISMISSED** [review]
One minor thing, but otherwise looks fine.

File: docs/source/changelog.rst (hunk 1 above)
Comment (gsheni) [reply docs/source/changelog.rst]: Done

Comment (gsheni) [reply docs/source/changelog.rst]: Eh, let's leave it for now.
https://github.com/alteryx/nlp_primitives/issues/85

Comment (gsheni) [reply docs/source/changelog.rst]: Actually done.
https://github.com/alteryx/nlp_primitives/pull/84/commits/1deb85a86fce48b0a922260657d04c726968ab14

**Review by rwedge — APPROVED** [review]

**Review by thehomebrewnerd — APPROVED** [review]

**Pull request merged**